guide/src/module.md

```diff
@@ -34,6 +34,9 @@ fn sum_as_string(a: i64, b: i64) -> String {
 
 The `#[pymodule]` procedural macro attribute takes care of exporting the initialization function of your module to Python. It can take as an argument the name of your module, which must be the name of the `.so` or `.pyd` file; the default is the Rust function's name.
 
+If the name of the module (the default being the function name) does not match the name of the `.so` or `.pyd` file, you may get an import error in Python with the following message:
+`ImportError: dynamic module does not define module export function (PyInit_name_of_your_module)`
```

Comment:
👍 Good idea to add this - happens to many beginners! I suggest changing "you may get an import error" to "you **will** get an import error". 😄